layouts/fastly/helix.vcl

```diff
@@ -630,16 +630,18 @@ sub hlx_type_blob {
 
   declare local var.sha STRING;
   declare local var.ext STRING;
+  declare local var.sas STRING;
 
   if (req.url.path ~ "^/hlx_(([0-9a-f]){40}).(jpg|jpeg|png|webp|gif)$") {
     set var.sha = re.group.1;
     set var.ext = req.url.ext;
+    set var.sas = table.lookup(secrets, "AZURE_BLOB_SAS_RO");
```

Comment:
what happens if the item is not in the dictionary?

Reply:
```suggestion
    set var.sas = table.lookup(secrets, "AZURE_BLOB_SAS_RO", "");
```

https://developer.fastly.com/reference/vcl/functions/table/table-lookup/

The third argument is the default return value.